src/validations/trainer.js

```diff
@@ -27,6 +28,40 @@ const validateCreation = (req, res, next) => {
   });
 };
 
+const validateUpdate = (req, res, next) => {
+  const trainerValidation = Joi.object({
+    firstName: Joi.string().min(3).max(20),
+    lastName: Joi.string().min(3).max(20),
+    dni: Joi.number().integer().min(100000).max(999999999),
+    phone: Joi.number().integer().min(1000000000).max(9999999999),
+    email: Joi.string().email().min(8).max(25)
+      .pattern(/^[^@]+@[^@]+\.[a-zA-Z]{2,}$/),
+    city: Joi.string().min(5).max(20),
+    password: Joi.string().min(4).max(25),
+    salary: Joi.number(),
```

Comment:
reading the first problematic, it said the trainers was paid for hour, maybe do a little change from salary to hourlyWage

Reply:
Hi, thanks for the review. The salary key is pre-defined in the planning document,